### PR TITLE
README: update listen address to new http address

### PR DESCRIPTION
**Pull request by chappjc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ random selection of the reliable nodes it knows about.
 To build and install from a checked-out repo, run `go install` in the repo's
 root directory.
 
-To start dcrseeder listening on udp 127.0.0.1:5354 with an initial connection to working testnet node 192.168.0.1:
+To start dcrseeder listening on udp localhost:8000 with an initial connection to working testnet node 192.168.0.1:
```
Comment (chappjc) [inline README.md:34]: hmm, this is tcp now too, right?

Comment (degeri) [inline README.md:34]:
```suggestion
To start dcrseeder listening on tcp localhost:8000 with an initial connection to working testnet node 192.168.0.1:
```

Comment (degeri) [reply README.md:34]: lol yeah just  commented that. since its http its tcp

**Review by davecgh — APPROVED** [review]

**Review by dajohi — APPROVED** [review]

**Comment by dajohi** [comment]
fixed in #40

**Pull request closed without merging**